test/lit.cfg

```diff
@@ -2428,8 +2428,12 @@ elif not kIsWindows and not run_os == 'wasi':
         lit_config.note('Testing with the just-built libraries')
 
     lit_config.note('Library load path: {0}'.format(os.path.pathsep.join(target_stdlib_path)))
+    env_path = "/usr/bin/env "
+    # Android doesn't have /usr/bin/, so just grab whatever `env` is in the PATH.
```

Comment:
It does have /bin/env which is pretty standard, why not use that?

Reply:
I'll try running the tests with that instead.

Reply:
Done